include/stdlib.h

```diff
@@ -27,6 +27,8 @@
 
 #define ATEXIT_MAX 32
 
+#define MB_CUR_MAX 4
```

Comment:
is this needed to be `4`? IMHO the real answer is `1`. Also, maybe define `MB_LEN_MAX` at the same time?

Reply:
Now I realised MB_CUR_MAX should be 1. MB_LEN_MAX might be 4?